meeting-notes/2020/Q1/2020-01-06--core-implementations-weekly.md

```diff
@@ -0,0 +1,216 @@
+  * Doesn't build on go 1.13
+  * Other issues...
+* Why not? Takes time and doesn't ship features.
+
+* js-ipfs - release a prerelease asap (hopefully this week)
+
+#### Upgrade Testing Infra/Process
+@jimpick / @stebalien / @daviddias
+
+* Holidays.
+* It works well enough for local testing, try it out!
+  * When you do, bug the testground team if you can't find documentation/examples.
+* Missing features
+  * Scaling (1k-10k nodes)
+  * Blocking inbound connections
+    * For now, you can simply _not listen_ for inbound connections and set an "address factory" to announce non-working external addresses.
+  * Per-subnet network controls (latency, bandwidth, etc.). Global controls work fine.
+  * Running multiple versions of ipfs/libp2p/dht, etc.
+
+#### Subdomain Gateway (Base32, Origin isolation)
+@lidel
+
+- we are finalizing new config options for subdomain (and other) gateways: [go-ipfs/pull/6096#thiscomment](https://github.com/ipfs/go-ipfs/pull/6096#issuecomment-569712890)
+  - next: get more eyeballs on it, then do go and js implementation
+
+#### Distributed Signaling (js-libp2p)
+@jacobheun
+
+- Oh hold pending the async refactor (will be starting up again soon)
+
+#### IPNS (go-ipfs / js-ipfs)
+@aschmahmann / @hugomrdias
+
+- Nothing new for now until planned DHT improvements are done
+
+#### Add Performance (go-ipfs / js-ipfs)
+@aschmahmann / @achingbrain
+
+- Go: Asynchronous Datastores PRs merged and BadgerDS defaults to asynchronous writes
+
+#### Migration to multihash keys in blockstore (js-ipfs)
+@alanshaw / @auhau
+
+- On hold
+
+#### Bitswap Updates
+@dirkmc
+
+- Currently working on porting server side of [Proof of Concept Bitswap changes](https://github.com/ipfs/go-bitswap/pull/189) to js-bitswap
+
+#### js-ipfs/js-libp2p/js-ipld async await refactor
+@achingbrain / @jacobheun / @vasco-santos / @alanshaw
+
+- js-libp2p
+  - Working towards an RC this week
+  - Finalizing examples and interop testing with go
+  - Integrating with js-ipfs
+  
+- js-ipfs
+  - Core interface test passing
+  - CLI tests passing
+  - HTTP API tests passing
+  - TODO: remaining core tests and swarm api (part of libp2p integration)
+
+#### UnixFS v1.5 (go-ipfs / js-ipfs)
+@achingbrain / @ribasushi
+- spec ( https://github.com/ipfs/specs/pull/236 )
+  - The placement of metadata has been finalized
+  - The content of the "POSIX meta" portion is still being refined, at about 95%
+- jsipfs implementation
+  - @achingbrain is working on a pilot at https://github.com/ipfs/js-ipfs-unixfs/pull/36 and others
+- goipfs implementation
+  - not yet started
+  
+  
+### Design Review Proposals
+> You want to propose something for design review.
+
+- @lidel: anyone interested in proposed config options for http gateways in go-ipfs (and js-ipfs) can comment in [go-ipfs/pull/6096#thiscomment](https://github.com/ipfs/go-ipfs/pull/6096#issuecomment-569712890) before we start implementations
+  - @stebalien
+  - @alanshaw
```

Comment:
@alanshaw could you review the final proposal in the linked comment and leave a comment?

Reply:
👍 done